src/freeze-dry/inline-styles.js

```diff
@@ -44,7 +44,7 @@ async function inlineLinkedStylesheets({rootElement, docUrl}) {
         } catch (err) {
             stylesheetText = '/* Oops! Freeze-dry failed to save this stylesheet. */'
         }
-        const styleEl = rootElement.ownerDocument.createElement('style')
+        const styleEl = rootElement.createElement('style')
```

Comment:
Why this change? `rootElement` should be an element, not a document. Perhaps check your tests if you are passing a document to it anywhere, and pass `document.documentElement` as rootElement instead.